cypress/integration/examples/hometask.e2e.js

```diff
@@ -0,0 +1,19 @@
+describe("Cypress e2e testing", () => {
+    it('should open page', () => {
+        cy.visit('http://localhost:4000/');
+
+        cy.title().should('include', "ReactJS App");
+    });
+
+    it('should has value', () => {
```

Comment:
Please update test description with information about the value (what value is this?)